lib/SourceMapDevToolPlugin.js

```diff
@@ -155,7 +160,9 @@ class SourceMapDevToolPlugin {
 					moduleToSourceNameMapping.set(module, sourceName);
 					usedNamesSet.add(sourceName);
 				}
-				tasks.forEach(task => {
+				tasks.forEach((task, index) => {
+					let progressFloat = (1.0 / tasks.length * index).toFixed(2);
```

Comment:
toFixed converts a number into a string. That's not correct here. Just pass `index / tasks.length` to reportProgress that's fine.